packages/xwayland.rb

```diff
@@ -1,49 +1,12 @@
 require 'package'
 
 class Xwayland < Package
-  description 'X server configured to (only) work with weston'
+  description 'The Xorg Server is the core of the X Window system.'
```

Comment:
Probably should mention something about Wayland here.

Reply:
Eh. It's just a meta-package that depends on xorg_server.